deploy/crds/compliance.openshift.io_complianceremediations_crd.yaml

```diff
@@ -40,6 +40,9 @@ spec:
               description: Whether the remediation should be picked up and applied
                 by the operator
               type: boolean
+            id:
+              description: A unique identifier of the remediation
```

Comment:
how is the id useful to administrators in general? Seems to me that this further ties the remediation object to the OpenSCAP schema

Reply:
In case you need to go to the arf output and learn more about the remediation, you can use the rule name.
If we ever support anything else than OpenScap profiles, we can put other data there. The ID is not a rule name, it just happens to contain the rule name if the remediation is generated by openscap. btw both the ID and the other fields are marked as `omitempty`.